predicates/path/path.go

```diff
@@ -0,0 +1,48 @@
+/*
+Package path implements a custom predicate to match routes
+based on the http path
+
+Examples:
+
+    example1: Path("/hello") -> "http://example.org";
+
+*/
+package path
+
+import (
+	"net/http"
+
+	"github.com/zalando/skipper/predicates"
+	"github.com/zalando/skipper/routing"
+)
+
+type predicate struct {
+	path string
+}
+type spec struct{}
+
+const name = "Path"
+
+// New creates a new Path predicate specification.
+func New() routing.PredicateSpec { return &spec{} }
+
+func (s *spec) Name() string {
+	return name
+}
+
+func (s *spec) Create(args []interface{}) (routing.Predicate, error) {
```

Comment:
I haven't checked, but am wondering what happens if this spec is not defined at all? The PathSubtree predicate works somehow.

Reply:
It's used here https://github.com/zalando/skipper/blob/79cbd3e1c15c438dfcaf0e79c7ee34dc232596b1/skipper.go#L1055

Reply:
Right. What I mean what if we remove it from there, too. The point would be to make it analogous to PathSubtree.

Another option would be to make PathSubtree analogous to the suggested one, I'm just not sure if we benefit from noop types if they are avoidable.

Reply:
Let's see if it works https://github.com/zalando/skipper/pull/1351/commits/499e5da57203c4b31ecb6ed5fcd054a4877918df